src/TodoList.jsx

```diff
@@ -1,27 +1,15 @@
-// /src/TodoList.jsx
-import * as React from 'react';
-import TodoListItem from './TodoListItem';
+import React from 'react';
 
-const todoList = [
-    { id: 1, title: 'Prueba1 Create an empty array and store it in a variable called todoList.' },
-    { id: 2, title: 'Create at least 3 objects with the required properties.' },
-    { id: 3, title: 'Insert a JavaScript expression in the unordered list.' },
-    { id: 4, title: 'Create a level one heading.' },
-    { id: 5, title: 'Create an ordered list.' },
-    { id: 6, title: 'Assign the todoList array in the JavaScript expression.' },
-    { id: 7, title: 'Return a list item (<li>) for each object in the array.' },    
-  ];
-//first part
-function TodoList() {
+function TodoList({ todoList }) {
   return (
     <div>
-      <ul> 
+      <ul>
         {todoList.map((todo) => (
-          <TodoListItem key={todo.id} props={todo} />
+          <li key={todo.id}>{todo.title}</li>
```

Comment:
This works fine but you already created a component that handles accessing and rendering your items.

Reply:
You're right, I already corrected it. `        {todoList.map((todo) => ( <TodoListItem key={todo.id} title={todo.title} />))}`

Reply:
I have a question, I know that I am learning React, to validate the forms do I have to do it with js or can I use html5 attributes? as "required" or do I have to do both validations? `      <form onSubmit={handleAddTodo}> 
        <label htmlFor="todoTitle">Fill: </label>
        <input type="text" id="todoTitle" name="todoTitle" value={todoTitle} onChange={handleTitleChange} required/>
        <button type="submit">Add Todo</button>
      </form> `
attribute requiered

Reply:
I'm not sure what you would use `trim()` for in your current code. It's also a string method so if you felt the need to remove whitespace from any user input you certainly could.